cacheops/redis.py

```diff
@@ -29,24 +30,48 @@ def handle_connection_failure(call):
 
 client_class_name = getattr(settings, 'CACHEOPS_CLIENT_CLASS', None)
 client_class = import_string(client_class_name) if client_class_name else redis.StrictRedis
+redis_replicas = []
+
+
+def ip(url):
+    return socket.gethostbyname(urlparse(url).hostname)
+
+
+def set_redis_replicas():
+    # the conf could be a list or string
+    # list would look like: ["redis://cache-001:6379/1", "redis://cache-002:6379/2"]
+    # string would be: "redis://cache-001:6379/1,redis://cache-002:6379/2"
+    global redis_replicas
+    if isinstance(settings.REDIS_REPLICAS, six.string_types):
+        temp = settings.REDIS_REPLICAS.split(',')
+    else:
+        temp = list(settings.REDIS_REPLICAS)
+    master_url = settings.REDIS_MASTER
+    master_ip = ip(master_url)
+    temp = [r for r in temp if ip(r) != master_ip]
+    temp = [r for r in temp for _ in range(settings.REDIS_REPLICA_WEIGHT)]
```

Comment:
@nicwolff 
I do not see any additional info on settings. REDIS_REPLICA_WEIGHT.
Is this something we forgot to initialize 
or coming from some additional work that needs to be done (need to mention this in the PR descr + "Do Not Merge" label + comment to the ticket if dependency on that)
or smth else?

Reply:
The settings come from Rover.